js/packages/components/settings/Home.tsx

```diff
@@ -90,9 +92,13 @@ export const Home: ScreenFC<'Settings.Home'> = withInAppNotification(
 		const selectedAccount = useSelector(selectSelectedAccount)
 		const ctx = useMessengerContext()
 
+		const dispatch = useDispatch()
```

Comment:
prefer using useAppDispatch and useAppSelector 👍

Reply:
Ok, and what does it bring ?